src/styles/reset.css

```diff
@@ -0,0 +1,14 @@
+body,
+h2 {
```

Comment:
In Colored blocks solution I explained why it is not a good idea to spread styles across multiple rule sets